apps/server/package.json

```diff
@@ -2,21 +2,26 @@
 	"name": "server",
 	"version": "1.0.0",
 	"description": "",
-	"main": "index.js",
+	"main": "dist/index.js",
 	"scripts": {
 		"build": "rm -rf dist && tsc",
-		"dev": "nodemon server.ts"
+		"dev": "nodemon"
+	},
+	"engines": {
+		"node": ">=18.0.0 <19.0.0"
```

Comment:
current LTS is 20. we shouldn't enforce earlier versions as mandatory.

Reply:
there's currently and [issue](https://github.com/TypeStrong/ts-node/issues/1997) with ts-node and ESM using node 20. I've tested node versions 18 and 19 and works fine.

Reply:
Right! I keep forgetting about that issue… and the more I come across it, the more I feel embracing ESM for our apps is premature.
Never mind, then 🤷